docs/model_pipeline.rst

```diff
@@ -102,6 +102,11 @@ A few more conventions are important to know:
   * such powered with electricity with :attr:`config.settings.optimize.el_key` followed by :attr:`carrier` and :attr:`region` (example: :attr:`{"electricity-heat_decentral-B": 1}`).
   * Do not provide :attr:`output_parameters` or leave their :attr:`var_value` empty to neglect a component in the constraint.
 
+* The capacity of specific components of a scenario can be expanded during an optimization, if the facade attribute `expandable` is set on `True`.
+
+  * To adjust this setting, the component needs to be specified in the file :attr:`component_attr_update.yml`.
+  * An overview of all facades, which includes `expandable` can be found in the facade attributes overview <https://oemof-tabular.readthedocs.io/en/latest/facades.html#>`_ provided by oemof.tabular.
+
```

Comment:
That needs to be adjusted according to what we learned from our conversation today. Thank you very much! :)

Reply:
Done with 2c010076dbf434b3d92a5c28bdae346091c55396